runtime/compiler/ilgen/Walker.cpp

```diff
@@ -6302,6 +6312,10 @@ TR_J9ByteCodeIlGenerator::loadStatic(int32_t cpIndex)
          handleSideEffect(treeTopNode);
          genTreeTop(treeTopNode);
          }
+      if (symbol->isFinal() && TR::TransformUtil::canFoldStaticFinalField(comp(), load))
```

Comment:
`canFoldStaticFinalField` returns yes/no/maybe could you make the comparison explicit for completeness.

Reply:
Good catch.

Reply:
Fixed